catroid/src/main/java/org/catrobat/catroid/content/bricks/SetLookBrick.java

```diff
@@ -231,7 +231,7 @@ public Brick clone() {
 
 	@Override
 	public List<SequenceAction> addActionToSequence(Sprite sprite, SequenceAction sequence) {
-		sequence.addAction(sprite.getActionFactory().createSetLookAction(getSprite(), look, wait));
+		sequence.addAction(sprite.getActionFactory().createSetLookAction(sprite, look, wait));
```

Comment:
I'm not sure if I understand you correctly.

`ProjectManager.getInstance().getCurrentScene().getSpriteList().get(0);` should always return the Background Sprite of the current scene, right? Can't I use that to access/change the background from *any other* Sprite?